ChopChop/Storage/Models/QuantityRecord.swift

```diff
@@ -9,6 +9,17 @@ extension QuantityRecord: Codable {
         case count, mass, volume, unit
     }
 
+    var type: BaseQuantityType {
+        switch self {
+        case .count:
+            return .count
+        case .mass(_, unit: _):
```

Comment:
I think the `(_, unit: _)` part might be unnecessary

Reply:
Ah yes will fix.